### PR TITLE
Fixed : "To merge the changes ( - Until March 31)." #66

### PR DESCRIPTION
**Pull request by daisukenishino**
# Fixes
https://github.com/OpenTouryoProject/OpenTouryoTemplates/milestone/3

### CODE REVIEW

**Comment by daisukenishino** [comment]
@yusukemaegawa 

Probably, preparation is completed.
I will propose a comparison with develop branch of /OpenTouryoProject/OpenTouryo/ repository for check of merge of this pull-request.

File: README.md [hunk 1]
```diff
@@ -22,7 +22,4 @@ Tutorial will be helpful to you for customizing template-base!)
```
Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

Please divide this file into English version and Japanese version.

Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

Please revise as follows:
- English version
    - "OpenTouryo" -> *OpenTouryo*
- Japanese version
    - "Open棟梁" -> **Open棟梁**

Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

Please write the usage for users who use Visual Studio 2013 or earlier.

**Comment by yusukemaegawa** [comment]
@daisukenishino 

[root_VS2015\README.md](https://github.com/OpenTouryoProject/OpenTouryoTemplates/blob/develop/root_VS2015/README.md)
[root_VS2015\Readme.ja.md](https://github.com/OpenTouryoProject/OpenTouryoTemplates/blob/develop/root_VS2015/Readme.ja.md)

It seems that Readme in OpenTouryoTemplates repository has been overwritten by Readme in OpenTouryo repository. But the contents of Readme in OpenTouryoTemplates repository differ from the one in OpenTouryo repository.

Please restore Readme files to original files.
Furthermore, please write about UWP_Sample and remove windows store app.

**Comment by yusukemaegawa** [comment]
@daisukenishino

I have finished checking. Please check the comments.

File: README.md (hunk 1 above)
Comment (daisukenishino) [inline README.md]: @yusukemaegawa 

I see.

Comment (daisukenishino) [inline README.md]: @yusukemaegawa

I see.

Comment (daisukenishino) [inline README.md]: @yusukemaegawa 

How about to write the link to the old version at below table?
https://github.com/OpenTouryoProject/OpenTouryoTemplates/blob/master/README.md#テンプレートベースtemplate-base

**Comment by daisukenishino** [comment]
@yusukemaegawa 

Is that so? If so, I will do so.
Is there no old point else besides that ?
Because, I made some modifications to root / README.

File: README.md (hunk 1 above)
Comment (yusukemaegawa) [inline README.md]: @daisukenishino 

How about guiding the users, who use Visual Studio 2013 or earlier, to use OpenTouryoTemplate 01-80?

For example:
When using Visual Studio 2013 or earlier, please use *[OpenTouryoTemplates version 01-80](https://github.com/OpenTouryoProject/OpenTouryoTemplates/tree/01-80)*.

Comment (daisukenishino) [inline README.md]: @yusukemaegawa

I see.

**Comment by daisukenishino** [comment]
@yusukemaegawa

Probably, preparation is completed.
- see : 
  - https://github.com/OpenTouryoProject/OpenTouryoTemplates/commit/84edb66c4b68fa9537a844372dd330cd72d3982e (Restore and change.)
  - [root-Readme.ja.xlsx](https://github.com/OpenTouryoProject/OpenTouryoTemplates/files/878124/root-Readme.ja.xlsx) (Extracted only the changed.)

Also, I made the following additional corrections to the material side.
- see : https://github.com/OpenTouryoProject/OpenTouryo/pull/197/commits/02ed550c12d79ace064c3e92f60265bf4ecf2c76

**Comment by daisukenishino** [comment]
@yusukemaegawa 

Please perform this merge after March 31st.

**Pull request merged**